.cirrus.yml

```diff
@@ -102,43 +102,6 @@ automation_task:
     always: *runner_stats
 
 
-# This task use to be called 'gating', however that name is being
```

Comment:
As a courtesy to future maintainers, could you remove `smoke` from line 49?

Reply:
Of course, good idea.